examples/src/test/java/jdocs/guide/ShoppingCartTags.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+// #guideTags
+package jdocs.guide;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ShoppingCartTags {
+  public static String Single = "shopping-cart";
+  public static List<String> Tags = Arrays.asList("carts-0", "carts-1", "carts-2");
```

Comment:
maybe use Java convention for constants: SINGLE, TAGS

Reply:
:+1: